typing/src/main/java/org/navgurukul/typingguru/utils/TypingGuruPreferenceManager.kt

```diff
@@ -0,0 +1,31 @@
+package org.navgurukul.typingguru.utils
+
+import android.content.Context
+import android.content.SharedPreferences
+
+object TypingGuruPreferenceManager {
```

Comment:
This should be class and should be provided by koin. As a general rule, we try to avoid object classes as much as possible to make our code easy to test